admin/app/clusters/[id]/LiveTables.tsx

```diff
@@ -163,6 +175,28 @@ export function LiveTables({
           noDataMessage="No services with function calls have been detected in the cluster lately."
         />
       </div>
+
+      <div className="mt-12">
+        {data.services.map((service) => (
```

Comment:
I have FE web experience 😆 , not sure if this is a reasonable approach or if there is another idiom, primitive etc for iterating over this data structure.

Reply:
No, this is the correct way to do it. Nice 🎉